Timer/Timer.cpp

```diff
@@ -491,7 +495,11 @@ namespace WPEFramework
                 dataToSend.length = sizeof(buf);
                 memcpy(dataToSend.data, buf, dataToSend.length);
                 LOGINFO("Timer send CEC %s", SLEEP == m_timerItems[timerId].mode ? "Standby" : "Wake");
-                IARM_Bus_Call(IARM_BUS_CECMGR_NAME,IARM_BUS_CECMGR_API_Send,(void *)&dataToSend, sizeof(dataToSend));
+                res=IARM_Bus_Call(IARM_BUS_CECMGR_NAME,IARM_BUS_CECMGR_API_Send,(void *)&dataToSend, sizeof(dataToSend));
```

Comment:
Declare res here instead of top (limits the scope)
IARM_Result_t res = IARM_Bus_Call(IARM_BUS_CECMGR_NAME,IARM_BUS_CECMGR_API_Send,(void *)&dataToSend, sizeof(dataToSend));